docs/content/installation/install_compatibility_requirements.md

```diff
@@ -51,13 +51,20 @@ Complete these steps for each worker node in Kubernetes cluster to prepare your
 
    `oc apply -f 99-ibm-attach.yaml`
 
-2. Configure storage system connectivity.
+2. Configure your storage system host attachment.
 
-    1.  Define the host of each Kubernetes node on the relevant storage systems with the valid WWPN (for Fibre Channel) or IQN (for iSCSI) of the node.
-
-    2.  For Fibre Channel, configure the relevant zoning from the storage to the host.
+    **Important:** The CSI driver does not define hosts on your storage system.
+    
+    Be sure to configure your storage system host attachment according to your storage system instructions.
 
-    3. Ensure proper connectivity.
+    The CSI driver supports the following connectivity for each worker node: Fibre Channel (WWPN), NVMe®/FC (NQN), and iSCSI (IQN).
+        
+    **Note:** 
+        - The CSI driver only supports NVMe®/FC on IBM Spectrum Virtualize family products.
+        - For Fibre Channel connectivity be sure that storage system is using one of the fully supported HBAs compatible with your host connection, as listed in the [IBM® System Storage® Interoperation Center (SSIC)](https://www-03.ibm.com/systems/support/storage/ssic/interoperability.wss).
+        - IBM DS8000 family storage systems only supports Fibre Channel connectivity.
+       
+    For more information, find your storage system documentation in [IBM Documentation](http://www.ibm.com/docs/).
 
 3. **For RHEL OS users:** Ensure that the following packages are installed.
```

Comment:
@ArbelNathan does nvme users need to install anything here?

Reply:
IDK, @roysahar?

Reply:
@rivkap / @oriyarde , sure they need, but we said that we point them to storage host attach section!

Reply:
@oriyarde  I agree - this should be included in any storage system requirements that we already say that they need to adhere to. (unless there is something special for our driver that they need but I highly doubt that)

Reply:
but our code uses the `nvme list` command, regardless of storage system requirements

Reply:
@roysahar  @ArbelNathan  does that mean that they need to do something special from our side?

Reply:
@rivkap / @oriyarde , if we will put the link from the storage there is a step to install the `nvme-cli`, see - [rhel-7.8-nvme-prerequisite](https://www.ibm.com/docs/en/flashsystem-7x00/8.4.x?topic=nhtrlos-attachment-requirements-hosts-that-are-running-linux-operating-system).

Reply:
yes, but we want our docs to remain correct even if the storage docs change...

Reply:
I don't "think they will", I just don't "think they won't", because they don't promise us to keep it as it is

Reply:
Certain things that are based off of storage system configurations need to be left to them. There are certain things that we need to make assumptions that it is covered sufficiently by the storage system documentation. If not, either they, or we, will get customer requests to update accordingly. We did our due diligence and made sure that the command itself is in SV documentation. There is no reason to assume that the customer won't know that it's necessary to implement (even if for some reason it's removed.)
I think that if this is the last issue, we should leave as-is (without adding the extra command) and close this documentation

@erantzabari

Reply:
> There is no reason to assume that the customer won't know that it's necessary to implement (even if for some reason it's removed.)

how would the customer know if it's not documented?

Reply:
why do we prefer to perform said due diligence for every storage system for which we add nvme support?
if this is only about time constraint, we could fix it in 1.9.0, it's not urgent

Reply:
> I think that if this is the last issue
[nope](https://github.com/IBM/ibm-block-csi-driver/pull/415#discussion_r769448127) (and I don't see how it is relevant to this conversation anyway)

Reply:
not every comment is a merge blocker. I provided my input. if you guys prefer the current way, so be it